metricbeat/scripts/mage/template/metricsetDoc.tmpl

```diff
@@ -11,6 +11,7 @@ This file is generated! See scripts/mage/docs_collector.go
 {{end -}}
 include::../../../{{.Mod.Path | basePath}}/{{.Mod.Base}}/{{.Metricset.Title}}/_meta/docs.asciidoc[]
 
+{{if .Metricset.IsDefault}}This is a default metricset: If its host module is unconfigured, it will be enabled by default.{{end}}
```

Comment:
Minor comment, but I would use a period instead of a colon here.

Reply:
Hah yah, I was on the fence about that colon.

Reply:
Colons can be very controversial. :-P